src/pageScript.ts

```diff
@@ -1,8 +1,30 @@
 import { VersionedTransaction } from "@solana/web3.js";
 import { parseTransaction } from 'viem';
```

Comment:
```suggestion
import { parseTransaction, Address } from 'viem';
```

Import Address from viem and use that as a type instead of `0x${string}`